pvt/helpers/src/models/tokens/Token.ts

```diff
@@ -49,7 +49,8 @@ export default class Token {
     if (this.symbol === 'WETH') {
       await token.deposit({ value: amount });
       await token.transfer(TypesConverter.toAddress(to), amount);
-    } else {
+    } else if (this.symbol !== 'wstETH' && this.symbol != 'wampl') {
+      // wstETH has the permissioned wrap()
```

Comment:
Not sure what this change is about. It seems odd that the call to `mint` would do nothing in this case.

Reply:
Because wampl has a privileged mint that has to be called by Admin, and in the test is done before "init," which results in minting all the tokens. (Otherwise the "Unbutton" tests fail.)

Reply:
Can't we remove the `mint` call then there instead of doing this?

Reply:
No, because it has to be done by the admin

Reply:
Ah that's annoying. Let's at least add a small comment here explaining that the unbutton tests do this, so we can reference it in the future.

Reply:
Will do -  I did try to avoid this, but every other configuration added complexity and was even worse